gcc/rust/typecheck/rust-tyty-call.cc

```diff
@@ -184,7 +184,10 @@ TypeCheckCallExpr::visit (FnType &type)
 		if ((int_ty.get_int_kind () == TyTy::IntType::IntKind::I8)
 		    || (int_ty.get_int_kind () == TyTy::IntType::IntKind::I16))
 		  {
-		    rust_error_at (arg_locus,
+		    rich_location richloc (line_table, arg_locus);
+		    richloc.add_fixit_replace (
+		      "cast the value to c_int: as c_int");
+		    rust_error_at (richloc, ErrorCode::E0617,
```

Comment:
Combining all of these into one function with different messages might seem like a bit overkill, but if it works better, I'll do that.